Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -67,8 +67,7 @@
  */
 class ConvertUrisImplementation extends AbstractFusionObject
 {
-    public const PATTERN_SUPPORTED_URIS
-        = '/(node|asset):\/\/([a-z0-9\-]+|([a-f0-9]){8}-([a-f0-9]){4}-([a-f0-9]){4}-([a-f0-9]){4}-([a-f0-9]){12})/';
+    private const PATTERN_SUPPORTED_URIS = '/(node|asset):\/\/([a-z0-9\-]+)/';
```

Comment:
That pattern is definitely more correct – but it basically copies the regex from `NodeAggregateId::PATTERN` and there's a risk, that this gets out of sync (as it did _before_).
Probably out of scope for this PR but since you changed the regex, we could think about constructing the pattern from the original regex (along the lines of `/(node):\/\/' . NodeAggregateId::PATTERN...`

Reply:
`NodeAggregateId::PATTERN` is private ... im fine with keeping it this way^^